### PR TITLE
Global global pattern

### PR DESCRIPTION
**Pull request by gallart**

### CODE REVIEW

File: fixtures/samples/infected/009.php [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<?php
+$ca2a = @$GLOBALS[$GLOBALS['k94124e4'][48].$GLOBALS['k94124e4'][
```
Comment (soullivaneuh) [inline fixtures/samples/infected/009.php]: The line should be finished.

```
Syntax error, unexpected end of file, expecting ']' on line 3.
```

**Review by soullivaneuh — CHANGES_REQUESTED** [review]

File: fixtures/samples/infected/009.php (hunk 1 above)
Comment (gallart) [reply fixtures/samples/infected/009.php]: Okay, I will ask hackers to write "clean code" ;)

Comment (soullivaneuh) [reply fixtures/samples/infected/009.php]: In this case, this is not a "clean code" case, but a not working at all code. :-)

Comment (gallart) [reply fixtures/samples/infected/009.php]: It's a sample from web shells

**Review by soullivaneuh — APPROVED** [review]

**Comment by soullivaneuh** [comment]
Please rebase and it should be OK.

**Pull request merged**